packages/tectonic-explorer/js/components/authoring.tsx

```diff
@@ -99,6 +102,7 @@ const AUTOCOMPLETE_OPTIONS: Record<string, any> = {
   sidebar: {
     "interactions": "Interactions",
     "metamorphism": "Metamorphism",
+    "sediments": "Sediments",
```

Comment:
This seems a little unrelated to the description of the change.  Is this needed?

Reply:
Right, I didn't mention that in the description, but it was intended. It should have been added before when I actually implemented the Sediments checkbox. It was impossible to configure it on the authoring page, and it was also breaking some logic that was trying to compare the current config value with defaults and update the URL if necessary. So, the authoring page was adding a sidebar option to the URL even when the author didn't modify defaults.